tools/check_languages.py

```
@@ -0,0 +1,53 @@
+import os, configparser
+
+dirty = False
+LANG_DIR = "./lang/"
+
+def open_lang(name):
+    parser = configparser.ConfigParser()
+    parser.optionxform = lambda str:str
```

Ruff (0.15.17)

[error] Lambda argument `str` is shadowing a Python builtin

(A006)